presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -183,6 +184,21 @@ private ThriftMetastoreUtil() {}
         return result;
     }
 
+    private static Map<String, String> toMetastoreApiTableParameters(Map<String, String> parameters)
```

Comment:
should it be done where new `Table` object is created instead? e.g `io.prestosql.plugin.hive.HiveMetadata#createTable`